components/pd_client/src/util.rs

```diff
@@ -41,6 +44,8 @@ pub struct Inner {
     >,
     pub hb_receiver: Either<Option<ClientDuplexReceiver<RegionHeartbeatResponse>>, Waker>,
     pub client_stub: PdClientStub,
+    address: String,
```

Comment:
It makes code more complicated. If it's only for testing purpose, I suggest to change to use other way.